aict_tools/configuration.py

```diff
@@ -162,6 +161,7 @@ class DispConfig:
     def __init__(self, config):
         model_config = config['disp']
 
+        self.coordinate_transformation = model_config['coordinate_transformation']
```

Comment:
We should check here: `if self.coordinate_transformation not in {'CTA', 'FACT'}:` ...